lib/browser-tests/puppeteer-helpers.js

```diff
@@ -49,19 +50,27 @@ export function withHmrcStylesAndScripts(body) {
       </script>
         ${body}
         <script src="/assets/hmrc-frontend-${version}.min.js" type="module"></script>
+        <script src="/assets/accessible-autocomplete-${version}.js" type="module"></script>
     </body>
     </html>
   `;
 }
 
 export async function render(page, body, options) {
   await page.setRequestInterception(true);
-  page.once('request', (req) => {
-    req.respond({ contentType: 'text/html', body });
-    page.setRequestInterception(false);
-  });
-  await page.goto('http://localhost:3000/', options); // if ever flaky, waitUntil networkidle0 (js loaded)
-  await page.evaluateHandle('document.fonts.ready');
-  await page.bringToFront();
+  const interceptPageRender = (req) => {
+    if (req.url() === 'http://localhost:3000/') {
+      return req.respond({ contentType: 'text/html', body });
+    }
+    return Promise.resolve().then(() => req.continue()).catch(() => {});
+  };
+  page.on('request', interceptPageRender);
+  try {
+    await page.goto('http://localhost:3000/', options); // if ever flaky, waitUntil networkidle0 (js loaded)
+    await page.bringToFront();
+  } finally {
+    page.off('request', interceptPageRender);
+    await page.setRequestInterception(false);
+  }
```

Comment:
I've no idea if these changes are needed either if the flakeyness was around that sampling of messages 🤷